app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class OrdersController < ApplicationController
+  before_action :authenticate_user!
+
+  def create
+    result = CreateOrder.new.call(cart: @cart, user: current_user)
+
+    if result.success?
+      flash[:notice] = result.value!
+      redirect_to root_path
+    else
+      flash[:notice] = result.failure
```

Comment:
to moglby byc raczej :alert :)

Reply:
poprawione - dla spójności również w carts_controller